### PR TITLE
[Windows] Add check usernames of vmtoolsd processes

### PR DESCRIPTION
**Pull request by Tomorrow9**
After user logon, there are 2 vmtoolsd processes, one username is "NT AUTHORITY\SYSTEM", the other is in "computer_hostname\logon_username" format.

### CODE REVIEW

File: windows/wintools_complete_install_verify/verify_vmtools.yml [hunk 1]
```diff
@@ -1,53 +1,73 @@
...
+- name: "Get problem device after VMware Tools install"
+  include_tasks: ../utils/win_get_problem_device.yml
 - name: "Check no problem device listed"
   ansible.builtin.assert:
     that:
       - gos_has_problem_device is defined
       - not gos_has_problem_device
-    fail_msg: "Problem devices were found on the system, please check listed problem devices: {{ gos_problem_device_list }}"
+    fail_msg: "Problem devices were found in guest, please check listed problem devices: {{ gos_problem_device_list }}"
+    success_msg: "No problem device is found in guest."
```
Comment (123lzxm) [inline windows/wintools_complete_install_verify/verify_vmtools.yml:73]: Is it better to use "in guest OS" in above fail and success messages?

Comment (Tomorrow9) [reply windows/wintools_complete_install_verify/verify_vmtools.yml]: Changed, thanks.

**Review by keirazhang — DISMISSED** [review]
LGTM.

**Review by 123lzxm — DISMISSED** [review]
LGTM, thanks.

**Review by keirazhang — APPROVED** [review]

**Pull request merged**